### ISSUE TITLE
docx->pdf converter problem with switch of orientation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#141]
```
A docx whic is composed with 3 pages : 

1) Page A with portrait orientation
1) Page B with landscape orientation
1) Page C with portrait orientation

is not well converted with docx->pdf converter :  the page B keep the portrait 
orientation from the page A.

See 
thirdparties-extension\org.apache.poi.xwpf.converter\src\test\resources\org\apac
he\poi\xwpf\converter\TestLandscapeFormat.docx
```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 13 Sep 2012 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the 1.0.0. See JUnit 
http://code.google.com/p/xdocreport/source/browse/thirdparties-extension/org.apa
che.poi.xwpf.converter.pdf/src/test/resources/org/apache/poi/xwpf/converter/core
/TestLandscapeFormat.docx
```

Original comment by `angelo.z...@gmail.com` on 2 Nov 2012 at 3:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `angelo.z...@gmail.com` on 2 Nov 2012 at 3:03
- Changed state: **Fixed**
